README.md

```diff
@@ -28,27 +28,31 @@ module:
 
 ## How to compile AddOn code
 
-### C code/Scripts, with GRASS source code on your computer
+### C code/Scripts, with GRASS GIS core source code on your computer
 
-Preparations (assuming the source code in $HOME/grass/):
-(if you have already built GRASS from source you don't need to do this
-again. If adding to a binary install, the versions must match exactly.
-You need to git checkout the exact tag or commit used for the binary.)
+Preparations (assuming the [GRASS GIS core source code](https://github.com/OSGeo/grass)
+being stored in `$HOME/grass/` - if you have already built GRASS GIS core from
+source code you don't need to do this again. If adding to a binary install,
+the versions must match exactly. You need to `git checkout` the exact tag
+or commit used for the binary.)
 
 ```
+# GRASS GIS core source code
 ./configure # [optionally flags]
 make libs
 ```
 
+### Compilation of GRASS GIS AddOns
```

Comment:
I find this subtitle a bit confusing... Doesn't this instruction correspond to C code/Scripts with GRASS core source code above? Perhaps add one more `#` so it's within the section?

Reply:
I think the heading should not be here unless all headings are restructured. The whole thing is part of the previous heading.

Reply:
Agreed and addressed in 6ee0ec088fe70f62862ae9222f640bdd16578ffd